test/groovy/DockerExecuteOnKubernetesTest.groovy

```diff
@@ -31,13 +33,13 @@ class DockerExecuteOnKubernetesTest extends BasePiperTest {
 
     @Rule
     public RuleChain ruleChain = Rules
-        .getCommonRules(this)
```

Comment:
Formatting changes only. I would like to suggest not to have formatting changes in such a PR in order not to blur the focus of the PR.